eng/pipelines/runtime-extra-platforms.yml

```diff
@@ -13,7 +13,7 @@ variables:
   - template: /eng/pipelines/common/variables.yml
 
 - ${{ if eq(variables.isNotSpecificPlatformOnlyBuild, true) }}:
```

Comment:
I don't think expand time conditions work for schedules. We would have to go to the AzDo UI for this pipeline and override the CI triggers/schedule.

Reply:
I don't think it'll work either.  

If you want such a behavior and not use AzDo UI, I think you'd have to make a template and then have two pipelines.  One with a schedule and one w/o.

Reply:
I disabled it from the UI.